### PR TITLE
fix: fix package's vulnerabilities with vite

### PR DESCRIPTION
**Pull request by BryanSeychelles**
## What & How
I done what i shoud do. I'm just kidding. I've init the projectt with vite and i got 0 vulnerabilities. JOB DONE.

## Tickets
* The link to the issue/ticket: _#3ayhg16

### CODE REVIEW

**Comment by Wbebey** [comment]
Task linked: [CU-3ayhg16 Repare package](https://app.clickup.com/t/3ayhg16)

File: frontend/dwaves-app/src/Pages/Login.tsx [hunk 1]
```diff
@@ -6,15 +6,14 @@ interface Props {
...
-
 export const ModalLogin: React.FC<Props> = ({ displayModal }) => {
     const [showLogin, setShowLogin] = useState(true)
 
     return (
         <div id='contain-modal'>
             <div id="modal">
                 <header>
-                    <img onClick={(e) => { displayModal(e) }} src={`${process.env.REACT_APP_URL}logo-dwaves.png`} alt="" />
+                    <img onClick={(e) => { displayModal(e) }} src={`http://localhost:3000/logo-dwaves.png`} alt="" />
```
Comment (TharickABDUL) [inline frontend/dwaves-app/src/Pages/Login.tsx:16]: no localhost:3000 at Dwaves !!

**Comment by TharickABDUL** [comment]
Thank you Bryan !

**Review by Wbebey — APPROVED** [review]
LGTM

**Review by Wbebey — APPROVED** [review]

**Pull request merged**